UI/ReflexFRP/mockLoginPage/mockClient/Main.hs

```diff
@@ -0,0 +1,104 @@
+{-# LANGUAGE ExplicitForAll, NoImplicitPrelude, NoMonomorphismRestriction #-}
+{-# LANGUAGE OverloadedStrings, RecursiveDo, ScopedTypeVariables          #-}
+{-# LANGUAGE TypeApplications, RecursiveDo                                             #-}
+
+{-# OPTIONS_GHC -fdefer-typed-holes #-}
+
+module Main where
+
+import ClassyPrelude
+import Data.Proxy
+import Reflex
+import Reflex.Dom
+import Servant.API
+import Servant.Reflex
+
+import MockAPI
+
+main = mainWidgetWithHead htmlHead body
+
+htmlHead :: forall t m. DomBuilder t m => m ()
+htmlHead = do
+  elAttr "meta" ("charset" =: "utf-8") (pure ())
+  elAttr "meta" ("name" =: "viewport" <> "content" =: "width=device-width, initial-scale=1.0") (pure ())
+  el "title" (text "ui-mockup")
+  styleSheet "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+  styleSheet "http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css"
+  styleSheet "http://localhost:8081/assets/css/Login-Form-Clean.css"
+  where
+    styleSheet addr = elAttr "link" ("rel"  =: "stylesheet" <> "href" =: addr) (return ())
```

Comment:
It doesn't seem so. Ryan told me that I should push the things that I deem useful (like the function below in the code). This probably would also be a worthy inclusion.